src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/SolutionPickerItemViewModel.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Windows.Input;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class SolutionPickerItemViewModel
+    {
+        public SolutionPickerItemViewModel(ICommand command, string filePath)
+        {
+            OpenSolutionCommand = command;
+            FullPath = filePath;
+            FilePath = Path.GetDirectoryName(filePath);
+            SolutionName = Path.GetFileName(filePath);
+        }
+
+        public ICommand OpenSolutionCommand { get; }
+
+        public string FilePath { get; }
+
+        public string SolutionName { get; }
+
+        public string FullPath { get; }
```

Comment:
Is this ever used? I think it is not. Please remove if not.

Reply:
It is used in the solutionpicker xaml, as a command parameter for the button